clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -488,11 +517,13 @@ test_template_with_repstr(void *dr_context)
                   << ", #stores: " << template_counts.stores << "\n";
         return 1;
     }
+    assert(syscall_stats.syscall_errno_counts.empty());
```

Comment:
Are `assert`s enabled in release build tests? I remember having to use our own macros in various tests for this reason.

Reply:
Good point.  I added this at the top before the headers:

```
// Enable asserts in release build testing too.
#undef NDEBUG
```